src/features/import/components/Upload/UploadFile.tsx

```diff
@@ -0,0 +1,177 @@
+import { UploadFileOutlined } from '@mui/icons-material';
+import { useDropzone } from 'react-dropzone';
+import {
+  Box,
+  Button,
+  CircularProgress,
+  IconButton,
+  Typography,
+} from '@mui/material';
+import { useCallback, useState } from 'react';
+
+import messageIds from '../../l10n/messageIds';
+import theme from 'theme';
+import useImportedFile from '../../hooks/useImportedFile';
+import { Msg, useMessages } from 'core/i18n';
+
+const UploadFile = () => {
+  const [error, setError] = useState<boolean>(false);
+  const messages = useMessages(messageIds);
+  const { parseData, loading } = useImportedFile();
+
+  const onDrop = useCallback((acceptedFiles: File[]): void => {
+    acceptedFiles.map((file: File) => {
+      parseData(file);
+    });
+  }, []);
+
+  const { getRootProps, getInputProps, open, isDragActive } = useDropzone({
+    accept: {
+      types: ['.csv', '.xls', '.xlsx'],
+    },
+    maxFiles: 1,
+    noClick: true,
+    noKeyboard: true,
+    onDrop,
+    onDropAccepted: () => setError(false),
+    onDropRejected: () => setError(true),
+    onError: () => {
+      setError(true);
+    },
+  });
+
+  return (
+    <>
+      <Box
+        {...getRootProps()}
+        p={12}
+        style={
+          loading
+            ? {
+                alignItems: 'center',
+                backgroundColor: '#1976D214',
+                borderColor: '#1976D2',
```

Comment:
I don't expect to see color codes like this in our code, because we have the theme colors from `useTheme()` (or in the hook created by `makeStyles()`) instead.